README.md

```diff
@@ -100,6 +100,71 @@ resources:
 Don't forget, you'll need to mount the directory containing the unix socket that
 the KMS server is listening on into the kube-apiserver.
 
+### Bootstrap during cluster creation
```

Comment:
Can you update this title to `### Bootstrap during cluster creation (kops)` ?

Reply:
done